src/order/complete_lattice.lean

```diff
@@ -70,8 +70,16 @@ class complete_lattice (α : Type*) extends bounded_lattice α, has_Sup α, has_
 
 /-- Create a `complete_lattice` from a `partial_order` and `Inf` function
 that returns the greatest lower bound of a set. Usually this constructor provides
-poor definitional equalities, so it should be used with
-`.. complete_lattice_of_Inf α _`. -/
+poor definitional equalities.  If other fields are known explicitly, they should be
+provided; for example, if `inf` is known explicitly, construct the `complete_lattice`
+instance as
+`instance : complete_lattice my_T :=
+ { inf := better_inf,
+   le_inf := ...,
+   inf_le_right := ...,
+   inf_le_left := ...
+   -- don't care to fix sup, Sup, bot, top
+   ..complete_lattice_of_Inf my_T _ }` -/
```

Comment:
````suggestion
```
instance : complete_lattice my_T :=
 { inf := better_inf,
   le_inf := ...,
   inf_le_right := ...,
   inf_le_left := ...
   -- don't care to fix sup, Sup, bot, top
   ..complete_lattice_of_Inf my_T _ }
```
-/
````

bors d+